application/modules/invoices/models/mdl_items.php

```diff
@@ -80,9 +80,9 @@ public function save($id = NULL, $db_array = NULL)
         $this->mdl_item_amounts->calculate($id);
 
         $this->load->model('invoices/mdl_invoice_amounts');
-        
-        if (isset($db_array->invoice_id)){
-            $this->mdl_invoice_amounts->calculate($db_array->invoice_id);
+
+        if (isset($db_array['invoice_id'])){
+            $this->mdl_invoice_amounts->calculate($db_array['invoice_id']);
```

Comment:
Ah yes, I see now that this method is called with an object in [application/modules/invoices/controllers/ajax.php](https://github.com/InvoicePlane/InvoicePlane/blob/master/application/modules/invoices/controllers/ajax.php#L48).

Sorry about that! :disappointed: